doc/changes/changes_1.1.0.md

```diff
@@ -0,0 +1,28 @@
+Exasol Row Level Security 1.1.0, released 2020-06-??.
+
+## Summary
+
+Group-based security is a new variant for protecting rows with RLS that we introduced in this release. You can define a group that a row belongs to and assign each user to one or more group. If a user is a member of the group, the row belongs to, that user can read the rows contents.
+
+Group-based security can be used alone or in combination with tenant-based security. In this case a user must either be the tenant who owns the row or a member of the row's group.
+
+Additionally we updated the dependencies and introduced dependency scanning.
+ 
+## Features / Enhancements
+ 
+* #42: Added Group based security
+* #45: Ported all tests to use the [test-db-builder](https://github.com/exasol/test-db-builder-java) 
+* #48: Updated dependencies, introduced dependency scan
+ 
+## Dependency updates
+ 
+* Added `org.sonatype.ossindex.maven` - `ossindex-maven-plugin` `3.1.0`
```

Comment:
I changed the pattern a little bit because I checked that the standard way to represent the group and artifact if is `group-id:artefact-id:version`. You can check updated template or my latest pull request in Virtual Schemas

Reply:
Adapted. Also added the header to the template.